viadot/tasks/genesys.py

```diff
@@ -74,6 +82,133 @@ def __call__(self, *args, **kwargs):
         """Download Genesys data to CSV"""
         return super().__call__(*args, **kwargs)
 
+    def merge_conversations_dfs(self, data_to_merge: list) -> DataFrame:
+        """Method to merge all the conversations data into a single data frame.
```

Comment:
Is it possible to describe what is happening inside this function? You have a lot of `normilize` and FORs. I will be easier to understand.

Reply:
I add and example and some more comments in the code, have a look at it!